metals/src/main/scala/scala/meta/internal/builds/SbtBuildTool.scala

```diff
@@ -98,9 +98,7 @@ case class SbtBuildTool(
       case Some(version) =>
         scribe.info(s"sbt ${version} found for workspace.")
         val valid = isCompatibleVersion(firstVersionWithBsp, version)
-        if (valid) {
-          writeSbtMetalsPlugins(workspace)
-        } else {
+        if (!valid) {
```

Comment:
We don't need to write the metals.sbt plugin file to generate the BSP config. And if we do we will forget to reload the build.